packages/api/amplify_api/test/ws/web_socket_auth_utils_test.dart

```diff
@@ -75,11 +80,38 @@ void main() {
       final payload =
           authorizedMessage.payload as SubscriptionRegistrationPayload;
 
-      _assertBasicSubscriptionPayloadHeaders(payload);
+      assertBasicSubscriptionPayloadHeaders(payload);
       expect(
         payload.authorizationHeaders[xApiKey],
         testApiKeyConfig.apiKey,
       );
     });
+
+    test('should generate an authorized message with custom authorizationMode',
```

Comment:
This shows user pools unless I'm missing something

Reply:
Yes, authorizes w user pools where the config's default mode is api key.